File: SingleAPIExample/python/StartInstallments.py
import splitit

username, password, api_key = (
    "username",
    "password",
    "api_key",
)

api_client = splitit.ApiClient(splitit.Configuration(username, password, api_key, sandbox=True))


def start_installments(ipn):
    resp = api_client.InstallmentPlanApi.installment_plan_start_installments(
        splitit.StartInstallmentsRequest(
            installment_plan_number=ipn,
        )
    )
    return resp

File: SingleAPIExample/java/StartInstallments.java
import com.splitit.*;
import com.splitit.sdk.model.*;
import java.math.BigDecimal;


public class StartInstallments {
    private String apiKey, username, password;
    public StartInstallments(String username, String password, String apiKey){
        this.username = username;
        this.password = password;
        this.apiKey = apiKey;
    }

    public void example(String ipn) throws ApiException{
        ApiClient apiClient = new ApiClient()
            .sandbox(true)
            .username(username)
            .password(password)
            .apiKey(apiKey);


        var result = apiClient.getInstallmentPlanApi().installmentPlanStartInstallments(
            new StartInstallmentsRequest()
            .installmentPlanNumber(ipn)
        );
        System.out.println(result);
    }
}
